milseed.js

```diff
@@ -0,0 +1,67 @@
+const mongoose = require('mongoose');
+
+mongoose.connect("mongodb://localhost/restaurant");
+
+const aboutSchema = mongoose.Schema({
+  id: {
+    type: Number,
+    unique: true,
+  },
+  name: String,
+  about: {
+    description: String,
+    hours: String,
+    price: String,
+    style: String,
+    phone: String,
+  },
+  banner: [],
+  photo: [],
+});
+
+
+const About = mongoose.model('About', aboutSchema);
+
+const create = (start, length) => {
+  const storage = [];
+  for (let i=start; i < start + length; i++) {
+    storage.push({
+      id: i,
+      name: 'String',
+      about: {
+        description: 'String',
+        hours: 'String',
+        price: 'String',
+        style: 'String',
+        phone: 'String',
+      },
+      banner: [],
+      photo: [],
+    });
+  }
+  return storage;
+}
+
+
+const seedDatabase = (creationFunction, creationAmountForEach, eachStartPoint, creationLimit) => {
+  About.count({}, (err, count) => {
+    if (count !== creationLimit) {
+      const amountOfRestaurants = creationFunction(eachStartPoint, creationAmountForEach);
+      console.log(amountOfRestaurants);
+      About.create(amountOfRestaurants)
+      .then(() => {
+        seedDatabase(create, creationAmountForEach, eachStartPoint + creationAmountForEach, creationLimit);
```

Comment:
seedDatabase takes in `creationFunction` but here you've hardcoded in `create`.  Also, is the recursion working okay?  It seems like it might be callstack heavy is all, so wondering if a while loop might be less memory intensive.